modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java

```diff
@@ -113,6 +98,41 @@ public boolean isShowBreadcrumb() {
 		return false;
 	}
 
+	private List<CET> _filterCETs(List<CET> cets, String type) {
+		Predicate<CET> filterPredicate = _getFilterPredicate(type);
+
+		if (filterPredicate == null) {
+			return cets;
+		}
+
+		return ListUtil.filter(cets, filterPredicate);
+	}
+
+	private Predicate<CET> _getFilterPredicate(String type) {
```

Comment:
Hi @thiago-buarqque, since this method is used only once inside another private method, can you inline it? Please consider other methods in the same situation on this pr. 
Thanks.

Reply:
@linolaoj  I kept this method because it will grown in the future for other types of client extensions. But I inlined the one that was calling it, the `_filterCETs`.